addons/web/static/src/js/views/form/form_controller.js

```diff
@@ -681,29 +681,33 @@ var FormController = BasicController.extend({
      * @param {OdooEvent} ev
      */
     _onOpenOne2ManyRecord: function (ev) {
+        var self = this;
         ev.stopPropagation();
         var data = ev.data;
         var record;
         if (data.id) {
             record = this.model.get(data.id, {raw: true});
         }
 
-        new dialogs.FormViewDialog(this, {
-            context: data.context,
-            domain: data.domain,
-            fields_view: data.fields_view,
-            model: this.model,
-            on_saved: data.on_saved,
-            on_remove: data.on_remove,
-            parentID: data.parentID,
-            readonly: data.readonly,
-            deletable: record ? data.deletable : false,
-            recordID: record && record.id,
-            res_id: record && record.res_id,
-            res_model: data.field.relation,
-            shouldSaveLocally: true,
-            title: (record ? _t("Open: ") : _t("Create ")) + (ev.target.string || data.field.string),
-        }).open();
+        this.model.mutex.getUnlockedDef()
```

Comment:
maybe add a one line comment to explain that you sync with the mutex to wait for potential onchanges